### ISSUE TITLE
NO_COLOR or --no-color options

### ISSUE THREAD

**Issue by sebastien** [zellij-org/zellij#3057]
`zellij` does not seem to support the `NO_COLOR` environment variable, or have a a `--no-color` option, which makes it hard to script the CLI as for instance:

```
LAST_SESSION="$(zellij list-sessions|head -n1|cut -d' ' -f1)"
echo "$LAST_SESSION"
zellij attach "$LAST_SESSION"
````

will fail with

```
No session with the name 'zippy-megalodon' found!
```

while

```
▷ zellij list-sessions
zippy-megalodon [Created 0s ago] (EXITED - attach to resurrect)
```

The reason is that `LAST_SESSION` contains the color codes. Now, if we do this instead, this will work.

```
LAST_SESSION="$(zellij list-sessions|sed 's/\x1B\[[0-9;]*m//g'|head -n1|cut -d' ' -f1)"
echo "$LAST_SESSION"
zellij attach "$LAST_SESSION"
```

TL;DR supporting `NO_COLOR` or `--no-color` would make scripting `zellij` with the shell much easier.

**Comment by cristiand391** [comment]
Even supporting `NO_COLOR`, change to the output could break scripts like these. I would prefer support for json output.

In the meantime, you should be using `zellij list-sessions --no-formatting --short` in your script.

**Comment by sebastien** [comment]
I didn't know there was a `--no-formatting` option (not listed in `--help`), but it does the trick. It would be nice to turn the `--no-formatting` flag on when `NO_COLOR` is set, but this solves my problem.

**Comment by qm3ster** [comment]
@sebastien https://no-color.org/ actually recommends that `NO_COLOR=1` don't even disable bold, underline, and italic, let alone positional formatting.
It should only remove color.
